templates/base/head_navbar.tmpl

```diff
@@ -8,7 +8,9 @@
 		</div>
 	</div>
 
-	{{if .IsSigned}}
+	{{if and .IsSigned .MustChangePassword}}
+		{{/* No links */}}
+	{{else if .IsSigned}}
```

Comment:
It could be just simplified to `if and .IsSigned (not .MustChangePassword)` to not need empty if

Reply:
I considered that and originally wrote that but thought it better to be explicit and for the if statement to match the next one. We'd only have to write a two line comment to explain.